doc/Writerside/topics/dataflow.md

```
@@ -0,0 +1,52 @@
+# Потоки данных
+
+Поток данных — это процесс передачи информации или данных между различными компонентами системы, начиная с источника 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)